dist/rev-sdk.d.ts

```diff
@@ -531,6 +545,29 @@ interface VbrickVideoEmbedConfig extends VbrickBaseEmbedConfig {
     forcedCaptions?: boolean;
     hideSettings?: boolean;
     hideFullscreen?: boolean;
+    /**
+     * shows video player with sidebar.
+     */
+    fullPlayer?: boolean;
```

Comment:
can this be `showFullPlayer` to remain consistent with `showFullWebcast` option for webcasts? If needed the config vs. embed parameters can be translated in `VbrickEmbed.ts` `getEmbedQuery()`